docs/usage/two-phase-transfer.md

```diff
@@ -0,0 +1,67 @@
+# Two-phase Transfer
+
+The name "two-phase transfer" is a reference to the [two-phase commit
+protocol for distributed
+transactions](https://en.wikipedia.org/wiki/Two-phase_commit_protocol).
+
+Single-phase transfers post funds to accounts immediately when they
+are created. That is, they affect the `debits_posted` and
+`credits_posted` fields on relevant accounts.
+
+In contrast to single-phase transfers, a two-phase transfer moves
+funds in stages:
+
+1. First, the pending transfer reserves funds. While reserved, they
+cannot be used by either the payer or payee. Only the `debits_pending`
+and `credits_pending` fields are modified on the relevant accounts at
+this point.
+
+2. Later, the application creates another transfer — with flags either
+marked `post_pending_transfer` or `void_pending_transfer`. The former
+moves all (or part) of the reserved funds to the destination; the
+latter reverts them to the original account.
```

Comment:
Since `post_pending_transfer`/`void_pending_transfer` are important and this is the first mention of them in this document, can you change them to link to the flags in the `transfers.md` reference document?

Also (unrelated), what do you think of:

2. Later, the application creates another transfer — with either of the following flags:
    - `post_pending_transfer`: Move all (or part) of the reserved funds to the destination.
    - `void_pending_transfer`: Revert all of the reserved funds to the original account.

Reply:
great!

Reply:
Added all fields as links.